src/vorta/views/archive_tab.py

```diff
@@ -954,3 +955,9 @@ def rename_result(self, result):
             self.populate_from_profile()
         else:
             self._toggle_all_buttons(True)
+
+    def toggle_compact_button_visibility(self):
```

Comment:
Minor idea that @real-yfprojects noted in another PR: If a button is disabled, it could give a reason. But just a nice-to-have from my side. It's fine as it is.